### PR TITLE
Add CHANGES.md

### PR DESCRIPTION
**Pull request by manics**
https://github.com/openmicroscopy/ansible-role-omero-server/blob/2.0.3/CHANGES.md has a list of breaking changes for the OMERO.server Ansible role. We should probably do the same for all important roles.

### CODE REVIEW

**Comment by sbesson** [comment]
Generally :+1: for communicating changes to the user-facing roles via some CHANGES/NEWS/CHANGELOG file. Is the expectation to maintain only the major changes or also improvements coming with minor/patch releases?

**Comment by manics** [comment]
The pattern I followed in the omero-server role was to state breaking changes **only**.

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Changes in Version 3
+
+## Summary of breaking changes
+
+- `postgresql.conf` is templated instead of making in-line modifications to the distribution configuration
+- `CONNECT` and `public` schema `USAGE` privileges are explicitly granted to database users
+- `postgresql_users_databases` has been replaced by two variables which are both required.
```
Comment (mtbc) [inline CHANGES.md]: This ends with a period but the others don't.

Comment (manics) [reply CHANGES.md]: Removed `.`

**Comment by joshmoore** [comment]
Thanks, @manics. There may need to be some unifications with the emerging `CHANGES` vs. `NEWS` policy as it settles.

**Pull request merged**